jaffle_shop/models/final/finance/fnl_finance_orders.sql

```diff
@@ -0,0 +1,15 @@
+WITH orders AS (
+    SELECT
+        *
```

Comment:
we usually encourage only necessary columns to be select as high up in he query as possible (but not a blocker)